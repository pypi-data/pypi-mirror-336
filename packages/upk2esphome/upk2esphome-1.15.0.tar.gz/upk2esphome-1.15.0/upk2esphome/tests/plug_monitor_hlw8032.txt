{
		"bt1_lv": 0,
		"bt1_pin": 7,
		"bt2_lv": 0,
		"bt2_pin": 24,
		"bt_type": 1,
		"ch_cddpid1": 9,
		"ch_cddpid2": 10,
		"ch_dpid1": 1,
		"ch_dpid2": 2,
		"ch_num": 2,
		"chip_type": 2,
		"jv": "1.1.1",
		"lose_vol": 0,
		"module": "WB2S",
		"net_trig": 0,
		"netled1_lv": 0,
		"netled1_pin": 8,
		"netn_led": 0,
		"nety_led": 0,
		"over_cur": 0,
		"over_vol": 0,
		"reset_t": 5,
		"rl1_lv": 1,
		"rl1_pin": 6,
		"rl2_lv": 1,
		"rl2_pin": 26,
		"rl_type": 0,
		"total_stat": 2,
		"vol_def": 0
	}
