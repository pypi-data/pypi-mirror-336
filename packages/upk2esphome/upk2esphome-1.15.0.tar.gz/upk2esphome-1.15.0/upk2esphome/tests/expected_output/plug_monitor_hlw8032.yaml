binary_sensor:
  - platform: gpio
    id: binary_switch_1
    pin:
      number: P7
      inverted: true
      mode: INPUT_PULLUP
    on_press:
      then:
        - switch.toggle: switch_1
  - platform: gpio
    id: binary_switch_2
    pin:
      number: P24
      inverted: true
      mode: INPUT_PULLUP
    on_press:
      then:
        - switch.toggle: switch_2

switch:
  - platform: gpio
    id: switch_1
    name: Relay 1
    pin: P6
  - platform: gpio
    id: switch_2
    name: Relay 2
    pin: P26

status_led:
  pin:
    number: P8
    inverted: true
